common/links-js.md

```diff
@@ -11,6 +11,7 @@
 [Object]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object "Object"
 [Promise]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise "Promise"
 [Readable]: https://nodejs.org/api/stream.html#stream_class_stream_readable "Readable"
+[ReadStream]: https://nodejs.org/api/fs.html#class-fsreadstream "ReadStream"
```

Comment:
their anchor tags are broken but thats not your fault. It does not jump to the right position for me.

Reply:
Yeah, I noticed that too, after a couple refreshes it does point to right location.